rtl_433_mqtt_autodiscovery/run.sh

```diff
@@ -25,6 +25,9 @@ fi
 if bashio::config.true "force_update"; then
   OTHER_ARGS="${OTHER_ARGS} --force_update"
 fi
+if bashio::config.true "debug"; then
+  OTHER_ARGS="${OTHER_ARGS} --debug"
+fi
```

Comment:
Did this get pulled in by mistake from the other PR?

Reply:
Shouldn't have been there either. I've applied it after creating the PR.